docs/HowTo/Get-Started/Use-Docker.md

```diff
@@ -28,6 +28,14 @@ Display the EthSigner command line help using the Docker image:
 !!! note
     `latest` runs the latest cached version. To pull the latest version, use `docker pull pegasyseng/ethsigner:latest`.
 
+You can specify [environment variables] with the docker image instead of the command line options.
+
+=== "example"
+
+    ```bash
+    docker run -e ETHSIGNER_CONFIG_FILE=/home/me/me_node/config.toml pegasyseng/ethsigner:latest file-based-signer
```

Comment:
I need to test it, but I suspect this particular example might need a volume mount i.e. /home/me/me_node should be mounted to a folder inside docker, then the config file env variable can be updated to point to mounted location. Similar to how password files are specified at line 55 `-v <~/myKeyFile>:/opt/ethsigner/keyfile`

Reply:
I'll update this to:

`docker run --mount type=bind,source=/home/me/me_node,target=/var/lib/ethsigner -e ETHSIGNER_CONFIG_FILE=/var/lib/ethsigner/config.toml pegasyseng/ethsigner:latest file-based-signer`